spec/features/managing_step_by_step_pages_spec.rb

```diff
@@ -280,6 +280,15 @@
     and_there_should_be_no_schedule_button
   end
 
+  scenario "Step by step has not been checked for broken links" do
+    given_there_is_a_draft_step_by_step_page
+    when_I_view_the_step_by_step_page
+    then_I_should_see_an_inset_prompt
+    and_the_prompt_should_link_to_the_check_links_page
```

Comment:
There isn't a "check links" page. What's happening at the moment is that a broken link report is being submitted. Perhaps the step split into two: 
```
and_I_see_a_prompt_to_check_for_broken_links
and_I_when_click_on_the_prompt_a_broken_link_report_is_submitted`
```
Though as stated above, I think this second step should just drop the user down to the steps section.

Reply:
Note to self to remove `and_the_prompt_should_link_to_the_check_links_page` from this commit.